test/e2e/specs/js.processors.js

```diff
@@ -0,0 +1,58 @@
+module.exports = {
+  'Basic JavaScript Processor test' : function (client) {
```

Comment:
Please have one tests per processor. (You can keep them in the same file. )
This way it'll be easier to find out what exactly got broken.